src/flavors/williams/static/js/views/app-view.js

```diff
@@ -20,6 +30,9 @@ const ActivityView = require("../../../../../base/static/js/views/activity-view"
 // END FLAVOR-SPECIFIC CODE
 const RightSidebarView = require("../../../../../base/static/js/views/right-sidebar-view");
```

Comment:
@goldpbear It looks like non of our custom-code flavors that extend AppView are working. I am getting this error:

>Module not found: Error: Can't resolve '../../../../../base/static/js/views/right-sidebar-view' in '/src/flavors/williams/static/js/views'

It seems like `right-sidebar-view` was deleted. Should we be porting `RightSidebarView` to the `StorySidebar` component, or are we no longer supporting these flavors?